cassowary/error.py

```diff
@@ -1,13 +1,17 @@
 from __future__ import print_function, unicode_literals, absolute_import, division
 
 
-class InternalError(Exception):
+class CassowaryException(Exception):
     pass
 
 
-class ConstraintNotFound(Exception):
+class InternalError(BaseException):
```

Comment:
Right. I did this change before the first coffee in the morning, sorry for that ;)